app/package.json

```
@@ -2,6 +2,10 @@
   "name": "app",
   "version": "0.1.0",
   "private": true,
+  "engines": {
+    "node": ">=18.0.0",
```

I think we could make this match the major version in the Dockerfile, which typically is the LTS version:

```suggestion
    "node": ">=20.0.0",
```